core-object.js

```diff
@@ -42,17 +42,22 @@ CoreObject.extend = function(options) {
 
       if (hasArgs(options.init)) {
         deprecation(
-          'Overriding init without calling this._super is deprecated. ' +
-            'Please call this._super(), addon: `' + options.name + '`'
+          'The addon `' + options.name + '` is overriding init without calling this._super. ' +
+            'This behaviour is deprecated. ' +
+            'This means that addon\'s writer needs to update it calling `this._super()` and release a new version. ' +
```

Comment:
Both this and the other deprection should make the same suggestion RE: `this._super.init && this._super.init.apply(this, arguments);` (the message here just says `this._super`).